pkg/selinux/selinux.go

```diff
@@ -269,20 +268,37 @@ func SelinuxGetEnforceMode() int {
 	return Disabled
 }
 
+func mcsPath(mcs string) string {
+	filename := fmt.Sprintf("/run/rkt/%s", mcs)
+	return filename
```

Comment:
I've added a variable to store this in the selinux module, and that's configured via an option passed to InitLabels at rkt runtime.